### PR TITLE
[UT] Add ut for utils/format.go

### PR DESCRIPTION
**Pull request by my-git9**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Ref https://github.com/karmada-io/karmada/issues/2579

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/cmdinit/utils/format_test.go [hunk 1]
```diff
@@ -0,0 +1,330 @@
+package utils
+
+import (
+	"net"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func stringInSlice(target string, str_array []string) bool {
```
Comment (XiShanYongYe-Chang) [inline pkg/karmadactl/cmdinit/utils/format_test.go:10]: To unify the parameter naming style of the entire repo, it is advised to use camel case naming.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm

File: pkg/karmadactl/cmdinit/utils/format_test.go (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply pkg/karmadactl/cmdinit/utils/format_test.go]: Lint error occurs here.

Comment (my-git9) [reply pkg/karmadactl/cmdinit/utils/format_test.go]: Updated. thanks

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**